boards/shields/wuxi_ov5640/wuxi_ov5640.overlay

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright 2024 NXP
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/{
+	chosen {
+		zephyr,camera = &csi;
+	};
+};
+
+&zephyr_cam_i2c {
+    status = "okay";
+
+    ov5640: ov5640@3c {
+        compatible = "ovti,ov5640";
+        reg = <0x3c>;
+        reset-gpios = <&gpio11 15 GPIO_ACTIVE_LOW>;
+        powerdown-gpios = <&gpio9 25 GPIO_ACTIVE_HIGH>;
+
+        port {
+            ov5640_ep_out: endpoint {
+                remote-endpoint = <&mipi_csi2rx_ep_in>;
+            };
+        };
+    };
+};
+
+&zephyr_mipi_csi {
```

Comment:
Should we use a name like `nxp_mipi_csi`? The properties being set here aren't generic across all MIPI-CSI controllers, are they?

Reply:
The properties here are generic. BTW, other boards / vendors can have (slightly) different camera pipeline, but if they are using "ov5640->mipi_csi->..." can reuse the overlay.

Reply:
Anyways, changed as your suggestion